spec.html

```diff
@@ -20688,35 +20687,50 @@ <h1>
-              <tr><td> `%`            </td><td> Number       </td><td> Number::remainder          </td></tr>
-              <tr><td> `+`            </td><td> Number       </td><td> Number::add                </td></tr>
-              <tr><td> `+`            </td><td> BigInt       </td><td> BigInt::add                </td></tr>
-              <tr><td> `-`            </td><td> Number       </td><td> Number::subtract           </td></tr>
-              <tr><td> `-`            </td><td> BigInt       </td><td> BigInt::subtract           </td></tr>
-              <tr><td> `&lt;&lt;`     </td><td> Number       </td><td> Number::leftShift          </td></tr>
-              <tr><td> `&lt;&lt;`     </td><td> BigInt       </td><td> BigInt::leftShift          </td></tr>
-              <tr><td> `&gt;&gt;`     </td><td> Number       </td><td> Number::signedRightShift   </td></tr>
-              <tr><td> `&gt;&gt;`     </td><td> BigInt       </td><td> BigInt::signedRightShift   </td></tr>
-              <tr><td> `&gt;&gt;&gt;` </td><td> Number       </td><td> Number::unsignedRightShift </td></tr>
-              <tr><td> `&amp;`        </td><td> Number       </td><td> Number::bitwiseAND         </td></tr>
-              <tr><td> `&amp;`        </td><td> BigInt       </td><td> BigInt::bitwiseAND         </td></tr>
-              <tr><td> `^`            </td><td> Number       </td><td> Number::bitwiseXOR         </td></tr>
-              <tr><td> `^`            </td><td> BigInt       </td><td> BigInt::bitwiseXOR         </td></tr>
-              <tr><td> `|`            </td><td> Number       </td><td> Number::bitwiseOR          </td></tr>
-              <tr><td> `|`            </td><td> BigInt       </td><td> BigInt::bitwiseOR          </td></tr>
-            </table>
-          </figure>
+          1. Let _operation_ be the abstract operation associated with _opText_ in the following table:
+            <figure>
+              <!-- emu-format ignore -->
+              <table class="lightweight-table">
+                <thead>
+                  <tr><th> _opText_       </th><th> _operation_                </th></tr>
+                </thead>
+                <tbody>
+                  <tr><td> `*`            </td><td> BigInt::multiply           </td></tr>
+                  <tr><td> `+`            </td><td> BigInt::add                </td></tr>
+                  <tr><td> `-`            </td><td> BigInt::subtract           </td></tr>
+                  <tr><td> `&lt;&lt;`     </td><td> BigInt::leftShift          </td></tr>
+                  <tr><td> `&gt;&gt;`     </td><td> BigInt::signedRightShift   </td></tr>
+                  <tr><td> `&amp;`        </td><td> BigInt::bitwiseAND         </td></tr>
+                  <tr><td> `^`            </td><td> BigInt::bitwiseXOR         </td></tr>
+                  <tr><td> `|`            </td><td> BigInt::bitwiseOR          </td></tr>
```

Comment:
Let's tighten up the whitespace padding on these columns.

Reply:
that'd be automatic if i remove the formatting ignore, but it looks much worse when i do that, which is why i left it in place from before

Reply:
No I don't mean all the way, I just mean the excess width where every column has too much.

Reply:
ah ok gotcha, on it